<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1259</width>
    <height>1251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>login</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>651</width>
     <height>1231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/login.jpg);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>643</x>
     <y>6</y>
     <width>701</width>
     <height>1241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(1, 111, 134);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>20</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#label {
	font: 26pt &quot;MS Shell Dlg 2&quot;;
text-align: center;
color: rgb(254, 208, 83);
margin: 0 auto;
font-size: 35px;
font-family: 'Droid Sans';
}

QLabel#label:hover {
	color: rgb(85, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Log in</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameBox">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>290</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#usernameBox {
	font: 16pt &quot;MS Shell Dlg 2&quot;;
padding-left: 20px;
padding-right: 20px;
border-radius: 20px;
background: #191970;
color: #fefefe;
}

QLineEdit#usernameBox:hover {
	background-color: rgb(42, 46, 52);
border-style: solid;
 border-width: 1px;
	border-color: rgb(245, 179, 1);
	color: rgb(241, 182, 88);
}

QLineEdit#usernameBox:pressed {
border-style: solid;
border-width: 1px;
	background-color: rgb(25, 69, 105);
	color: rgb(85, 255, 255);
}</string>
   </property>
   <property name="placeholderText">
    <string>username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordBox">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>420</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#passwordBox {
	font: 16pt &quot;MS Shell Dlg 2&quot;;
padding-left: 20px;
padding-right: 20px;
border-radius: 20px;
background: #191970;
color: #fefefe;
}

QLineEdit#passwordBox:hover {
	background-color: rgb(42, 46, 52);
border-style: solid;
 border-width: 1px;
	border-color: rgb(245, 179, 1);
	color: rgb(241, 182, 88);
}

QLineEdit#passwordBox:pressed {
border-style: solid;
border-width: 1px;
	background-color: rgb(25, 69, 105);
	color: rgb(85, 255, 255);
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forgetpassbtn">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>610</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#forgetpassbtn {
	font: 16pt &quot;MS Shell Dlg 2&quot;;
Padding: 1px;
Border-radius: 5px;
background-color: rgb(241, 182, 88);
color: rgb(25, 69, 105);
}

QPushButton#forgetpassbtn:hover {
	background-color: rgb(42, 46, 52);
border-style: solid;
 border-width: 1px;
	color: rgb(241, 182, 88);
}

QPushButton#forgetpassbtn:pressed {
border-style: solid;
border-width: 1px;
	border-color: rgb(25, 69, 105);
	color: rgb(85, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Forget password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>700</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton#loginButton {
color: white;
border-radius: 5px;
Background: #393;
color: #fefefe;
font: 16pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton#loginButton:hover {
	background-color: rgb(42, 46, 52);
border-style: solid;
 border-width: 1px;
	color: rgb(241, 182, 88);
}

QPushButton#loginButton:pressed {
border-style: solid;
border-width: 1px;
	background-color: rgb(25, 69, 105);
	color: rgb(85, 255, 255);
}
</string>
   </property>
   <property name="text">
    <string>sign in</string>
   </property>
  </widget>
  <widget class="QPushButton" name="regButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>700</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#regButton {
color: white;
border-radius: 5px;
Background: #933;
color: #fefefe;
font: 16pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton#regButton:hover {
	background-color: rgb(42, 46, 52);
border-style: solid;
 border-width: 1px;
	color: rgb(241, 182, 88);
}

QPushButton#regButton:pressed {
border-style: solid;
border-width: 1px;
	background-color: rgb(25, 69, 105);
	color: rgb(85, 255, 255);
}
</string>
   </property>
   <property name="text">
    <string>sign up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rules">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>900</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#rules {
	font: 16pt &quot;MS Shell Dlg 2&quot;;
Padding: 1px;
Border-radius: 5px;
background-color: rgb(241, 182, 88);
color: rgb(25, 69, 105);
}

QPushButton#rules:hover {
	background-color: rgb(42, 46, 52);
border-style: solid;
 border-width: 1px;
	color: rgb(241, 182, 88);
}

QPushButton#rules:pressed {
border-style: solid;
border-width: 1px;
	border-color: rgb(25, 69, 105);
	color: rgb(85, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>rules</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>900</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#exit {
	font: 16pt &quot;MS Shell Dlg 2&quot;;
Padding: 1px;
Border-radius: 5px;
background-color: rgb(241, 182, 88);
color: rgb(25, 69, 105);
}

QPushButton#exit:hover {
	background-color: rgb(42, 46, 52);
border-style: solid;
 border-width: 1px;
	color: rgb(241, 182, 88);
}

QPushButton#exit:pressed {
border-style: solid;
border-width: 1px;
	border-color: rgb(25, 69, 105);
	color: rgb(85, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_showpass">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>530</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox#checkBox_showpass {
	font: 75 8pt &quot;MS Shell Dlg 2&quot;;
padding-left: 10px;
border-radius: 5px;
color: #fefefe;
}

QCheckBox#checkBox_showpass:hover {
	color: rgb(241, 182, 88);
}

QCheckBox#checkBox_showpass:pressed {
	color: rgb(85, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Show my password</string>
   </property>
  </widget>
  <widget class="QLabel" name="loginLabel">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>190</y>
     <width>561</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #fefefe;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
